### PR TITLE
Add ETH estimates

### PR DESCRIPTION
**Pull request by pcothenet**
### What

- Add ETH estimates

### Why

- Proof of stake >> Proof of work

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [x] Have you built the package locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version?
- [x] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: src/api/ProjectsApi.js [hunk 1]
```diff
@@ -14,13 +14,11 @@ export default class ProjectsApi {
...
-      this
-    );
+    this.retrieveProjectWithHttpInfo =
+      this.retrieveProjectWithHttpInfo.bind(this);
     this.retrieveProjects = this.retrieveProjects.bind(this);
-    this.retrieveProjectsWithHttpInfo = this.retrieveProjectsWithHttpInfo.bind(
-      this
-    );
+    this.retrieveProjectsWithHttpInfo =
+      this.retrieveProjectsWithHttpInfo.bind(this);
```
Comment (pcothenet) [inline src/api/ProjectsApi.js:21]: It looks like there was a problem with my prettier settings yesterday and the code formatting hadn't run properly. Sorry about the noise here!

File: test/integration/estimates.test.js [hunk 2]
```diff
@@ -67,16 +67,50 @@ describe('Estimates Integration', function () {
...
+        gas_used: 1000
+      }
+    );
+
+    const estimate = createEstimateResponse.data;
+
+    expect(estimate.type).to.be.eq('ethereum');
+    expect(estimate.mass_g).to.be.above(0);
+    expect(estimate.production).to.be.eq(false);
+  });
```
Comment (kleinjm) [inline test/integration/estimates.test.js:115]: there doesn't seem to be a test for the `project_id` param. Is that a concern?

**Review by kleinjm — APPROVED** [review]
No blockers but...
The test failures on this branch seem to be intermittently failing. I reran the GH actions and saw different errors on different node versions. I pulled this branch locally (toggling builds with `nvm`) and saw the same behavior where tests would fail randomly. Unfortunately, I couldn't reproduce stably nor do I see an obvious issue with the most common failure; `createEthereumEstimate` is `undefined`. I'm unsure if this is a common issue so didn't keep digging but let me know if you'd like to take a look together

**Comment by pcothenet** [comment]
> No blockers but...
> The test failures on this branch seem to be intermittently failing. I reran the GH actions and saw different errors on different node versions. I pulled this branch locally (toggling builds with `nvm`) and saw the same behavior where tests would fail randomly. Unfortunately, I couldn't reproduce stably nor do I see an obvious issue with the most common failure; `createEthereumEstimate` is `undefined`. I'm unsure if this is a common issue so didn't keep digging but let me know if you'd like to take a look together

Yeah, I think one of the issue is a race condition between the tests on each version of node. It looks like the preference test  can have conflicts if several tests are run in a matrix. Let me add that to Asana.

**Pull request merged**